### PR TITLE
WP 6.2 compatibility update

### LINKED ISSUE #141: The plugin hasn't been tested with the latest version of WordPress
There is a new WordPress version that the plugin hasn't been tested with. Please test it and then change the "Tested up to" field in the plugin readme.

**Tested up to:** 6.1
**Latest version:** 6.2

This issue will be closed automatically when the versions match.

### PR DESCRIPTION
**Pull request by jayedul**
### Description of the Change
WP compatibility version upgraded to 6.2 and tests passed successfully.

Closes #141

### Changelog Entry

> Changed - WP compatibility version upgraded to 6.2

### Credits

Props @jayedul 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jayedul** [comment]
Though it is WP 6.2 compatibility, I had to update cypress test script and utils library to pass the Github checks.

File: tests/cypress/integration/pmk-block-patterns.test.js [hunk 1]
```diff
@@ -12,12 +12,13 @@ describe('Check if Media Kit Block Pattern is available for use', () => {
...
+            } else if ( $body.find( '[aria-label="Publisher Media Kit"]' ).length > 0 ) {
+				cy.get('[aria-label="Publisher Media Kit"]').click();
+			}
         });
 
-        cy.get(
-            '[aria-label="Publisher Media Kit - Cover"] .block-editor-block-patterns-list__item-title',
-        ).click({ force: true });
+        cy.get('[aria-label="Publisher Media Kit - Cover"]').click();
+
```
Comment (faisal-alvi) [inline tests/cypress/integration/pmk-block-patterns.test.js]:
```suggestion

```

File: tests/cypress/integration/pmk-block-patterns.test.js [hunk 2]
```diff
@@ -12,12 +12,13 @@ describe('Check if Media Kit Block Pattern is available for use', () => {
                 cy.get('.components-select-control__input').select('publisher-media-kit', {
                     force: true,
                 });
-            }
+            } else if ( $body.find( '[aria-label="Publisher Media Kit"]' ).length > 0 ) {
```
Comment (faisal-alvi) [inline tests/cypress/integration/pmk-block-patterns.test.js]:
```suggestion
            } else if ($body.find( '[aria-label="Publisher Media Kit"]' ).length > 0) {
```

File: tests/cypress/integration/pmk-block-patterns.test.js [hunk 3]
```diff
@@ -12,12 +12,13 @@ describe('Check if Media Kit Block Pattern is available for use', () => {
...
+				cy.get('[aria-label="Publisher Media Kit"]').click();
+			}
```
Comment (faisal-alvi) [inline tests/cypress/integration/pmk-block-patterns.test.js]:
```suggestion
                        cy.get('[aria-label="Publisher Media Kit"]').click();
            }
```

Comment (faisal-alvi) [reply tests/cypress/integration/pmk-block-patterns.test.js]: In all the test files, the spaces are used, so we should follow the same here.

File: tests/cypress/integration/pmk-block-patterns.test.js (hunk 2 above)
Comment (faisal-alvi) [reply tests/cypress/integration/pmk-block-patterns.test.js]: For the sake of code consistency.

**Review by faisal-alvi — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/pmk-block-patterns.test.js [hunk 4]
```diff
@@ -12,12 +12,12 @@ describe('Check if Media Kit Block Pattern is available for use', () => {
                 cy.get('.components-select-control__input').select('publisher-media-kit', {
                     force: true,
                 });
+            } else if ($body.find( '[aria-label="Publisher Media Kit"]' ).length > 0) {
+                        cy.get('[aria-label="Publisher Media Kit"]').click();
             }
```
Comment (faisal-alvi) [inline tests/cypress/integration/pmk-block-patterns.test.js]:
```suggestion
            } else if ($body.find( '[aria-label="Publisher Media Kit"]' ).length > 0) {
                        cy.get('[aria-label="Publisher Media Kit"]').click();
            }
```

**Review by faisal-alvi — APPROVED** [review]

**Pull request merged**